src/components/common/ButtonComponent.jsx

```diff
@@ -3,17 +3,17 @@ import { Button } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
 const ButtonComponent = ({
-  buttonName, color, handleClick, buttonState
+  buttonName, color, handleClick, buttonState, customCss
 }) => {
   if (color === 'primary') {
     return (
-      <Button primary onClick={handleClick} loading={buttonState}>
+      <Button primary onClick={handleClick} loading={buttonState} id={customCss}>
```

Comment:
Why not class instead of id?

Reply:
It seems there was something I had not done with specificity. I got working with class.